master/buildbot/db/users.py

```diff
@@ -28,21 +28,23 @@ class UsDict(dict):
 class UsersConnectorComponent(base.DBConnectorComponent):
     # Documentation is in developer/db.rst
 
+    def __init__(self, *args, **kwargs):
+        super(UsersConnectorComponent, self).__init__(*args, **kwargs)
+        self._tbl = self.db.model.users
```

Comment:
Yes, it was mainly the code duplication I was referring to -- which has been addressed here.  We do sometimes use local variables for tables, but not class variables, so I agree with Pierre that this particluar hunk, and uses of `self._tbl*`, should not be merged.